### ISSUE TITLE
Lookup of new/changed contacts doesn't happen

### ISSUE THREAD

**Issue by w1n5l0w** [nextcloud/maps#104]
Expectation: Lookup process of new/changed contacts shoud be triggered automatically with contact hooks and background jobs with an immediate effect.

Problem: The only change I experience is that changed contacts disappear from being displayed and also the total number of contacts reduces according to the number of changed contacts. A reload of the app page doesn't make a difference, disabling an un/reinstalling the app doesn't change anything. Experiance is that after first installation the Maps DB is populated with all contacts at the time and changed contacts disappear from Maps, changed or new contacts won't be listed or displayed on Maps. This doen*t change with regular cron jobs, site reloads, even after days.

When you need more info on setup, let me know.

**Comment by eneiluj** [comment]
Hey, thanks for the bug report.

Manual contact edition is supposed to have an immediate effect on its location. Maps catches the contact change event and tries to resolve the new address.

When a lot of contact addresses have to be resolved, there is a minimum time of 1 second between Nominatim queries to avoid being blacklisted.

My guess is that you have many contacts and Maps still not had enough time to resolve it all.

Could you check the last row of `maps_address_geo` table just after you changed an address manually? It would help to know if the vCard change event was caught correctly.

**Comment by Schmuuu** [comment]
Hello,

First of all: Awesome app and a huge thank you for developing this! 

While I have the issue that no contacts at all are shown in this app I was searching for already open issues and I'm not sure if my issue is related to this one here.

I updated contacts and had a look at the database table you asked to check. All my address changes are successfully updated in that table. 

What I'm wondering about: might there be an issue with german letters?
The table contains for example the following address
```
yyyy-zzzz-Straße 200;München;Bayern;8XXXX;Deutschland
``` 

I noticed error messages in the nextcloud log as well:
```
{"reqId":"fb0IuxP793Mdu5zKZHp5","level":3,"time":"2019-09-01T02:03:27+02:00","remoteAddr":"<IP-ADDR>","user":"<USER1>","app":"index","method":"GET","url":"\/apps\/maps\/contacts","message":{"Exception":"Doctrine\\DBAL\\Exception\\DriverException","Message":"An exception occurred while executing 'INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)' with params [\";;Tessiner Strasse 96;Rostock;;1XXXX;\", \";;tessinerstrasse96;rostock;;1XXXX;\", \"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf\",\"XX.XXXXXXX\", \"XX.XXXXXXXXXXXX\", true]:\n\nSQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'object_uri' at row 1","Code":0,"Trace":[{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/DBALException.php","line":184,"function":"convertException","class":"Doctrine\\DBAL\\Driver\\AbstractMySQLDriver","type":"->","args":["An exception occurred while executing 'INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)' with params [\";;Tessiner Strasse 96;Rostock;;1XXXX;\", \";;tessinerstrasse96;rostock;;1XXXX;\", \"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf\", \"XX.XXXXXXX\", \"XX.XXXXXXXXXXXX\", true]:\n\nSQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'object_uri' at row 1",{"errorInfo":["22001",1406,"Data too long for column 'object_uri' at row 1"],"__class__":"Doctrine\\DBAL\\Driver\\PDOException"}]},{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/DBALException.php","line":158,"function":"wrapException","class":"Doctrine\\DBAL\\DBALException","type":"::","args":[{"__class__":"Doctrine\\DBAL\\Driver\\PDOMySql\\Driver"},{"errorInfo":["22001",1406,"Data too long for column 'object_uri' at row 1"],"__class__":"Doctrine\\DBAL\\Driver\\PDOException"},"An exception occurred while executing 'INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)' with params [\";;TessinerStrasse 96;Rostock;;1XXXX;\", \";;tessinerstrasse96;rostock;;1XXXX;\", \"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf\", \"XX.XXXXXXX\", \"XX.XXXXXXXXXXXX\", true]:\n\nSQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'object_uri' at row 1"]},{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Connection.php","line":1088,"function":"driverExceptionDuringQuery","class":"Doctrine\\DBAL\\DBALException","type":"::","args":[{"__class__":"Doctrine\\DBAL\\Driver\\PDOMySql\\Driver"},{"errorInfo":["22001",1406,"Data too long for column 'object_uri' at row 1"],"__class__":"Doctrine\\DBAL\\Driver\\PDOException"},"INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)",{"1":";;Tessiner Strasse 96;Rostock;;1XXXX;","2":";;tessinerstrasse96;rostock;;1XXXX;","3":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","4":"XX.XXXXXXX","5":"XX.XXXXXXXXXXXX","6":true}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/DB\/Connection.php","line":216,"function":"executeUpdate","class":"Doctrine\\DBAL\\Connection","type":"->","args":["INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)",[";;Tessiner Strasse 96;Rostock;;1XXXX;",";;tessinerstrasse96;rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","XX.XXXXXXX","XX.XXXXXXXXXXXX",true],[2,2,2,2,2,5]]},{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Query\/QueryBuilder.php","line":222,"function":"executeUpdate","class":"OC\\DB\\Connection","type":"->","args":["INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`,`looked_up`) VALUES(:dcValue9, :dcValue10, :dcValue11, :dcValue12, :dcValue13, :dcValue14)",{"dcValue1":"d784cbd5-17a2-4283-96b0-a55ae754694d.vcf","dcValue2":";;th\u00e4lmann-stra\u00dfe;rostock;mecklenburg-vorpommern;1XXXX;deutschland","dcValue3":"59C5806D-156E-4D72-B7D2-48189AB30237.vcf","dcValue4":";;yyy-zzz-stra\u00dfe50;m\u00fcnchen;bayern;8XXXX;deutschland","dcValue5":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","dcValue6":";;tessinerstrasse96;rostock;;1XXXX;","dcValue7":true,"dcValue8":";;tessinerstrasse96;rostock;;1XXXX;","dcValue9":";;Tessiner Strasse 96;Rostock;;1XXXX;","dcValue10":";;tessinerstrasse96;rostock;;1XXXX;","dcValue11":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","dcValue12":"XX.XXXXXXX","dcValue13":"XX.XXXXXXXXXXXX","dcValue14":true},{"dcValue1":2,"dcValue2":2,"dcValue3":2,"dcValue4":2,"dcValue5":2,"dcValue6":2,"dcValue7":5,"dcValue8":2,"dcValue9":2,"dcValue10":2,"dcValue11":2,"dcValue12":2,"dcValue13":2,"dcValue14":5}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/DB\/QueryBuilder\/QueryBuilder.php","line":214,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":269,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":110,"function":"scheduleForLookup","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":58,"function":"lookupAddress","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Controller\/ContactsController.php","line":93,"function":"addressToGeo","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":166,"function":"getContacts","class":"OCA\\Maps\\Controller\\ContactsController","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Maps\\Controller\\ContactsController"},"getContacts"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/App.php","line":126,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Maps\\Controller\\ContactsController"},"getContacts"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Routing\/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Maps\\Controller\\ContactsController","getContacts",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"_route":"maps.contacts.getContacts"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"_route":"maps.contacts.getContacts"}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/Route\/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"_route":"maps.contacts.getContacts"}]},{"file":"\/var\/www\/nextcloud\/lib\/base.php","line":975,"function":"match","class":"OC\\Route\\Router","type":"->","args":["\/apps\/maps\/contacts"]},{"file":"\/var\/www\/nextcloud\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Driver\/AbstractMySQLDriver.php","Line":125,"Previous":{"Exception":"Doctrine\\DBAL\\Driver\\PDOException","Message":"SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'object_uri' at row 1","Code":"22001","Trace":[{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Connection.php","line":1079,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDOStatement","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/lib\/private\/DB\/Connection.php","line":216,"function":"executeUpdate","class":"Doctrine\\DBAL\\Connection","type":"->","args":["INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)",[";;Tessiner Strasse 96;Rostock;;1XXXX;",";;tessinerstrasse96;rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","XX.XXXXXXX","XX.XXXXXXXXXXXX",true],[2,2,2,2,2,5]]},{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Query\/QueryBuilder.php","line":222,"function":"executeUpdate","class":"OC\\DB\\Connection","type":"->","args":["INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(:dcValue9, :dcValue10, :dcValue11, :dcValue12, :dcValue13, :dcValue14)",{"dcValue1":"d784cbd5-17a2-4283-96b0-a55ae754694d.vcf","dcValue2":";;th\u00e4lmann-stra\u00dfe;rostock;mecklenburg-vorpommern;1XXXX;deutschland","dcValue3":"59C5806D-156E-4D72-B7D2-48189AB30237.vcf","dcValue4":";;yyy-zzz-stra\u00dfe50;m\u00fcnchen;bayern;8XXXX;deutschland","dcValue5":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","dcValue6":";;tessinerstrasse96;rostock;;1XXXX;","dcValue7":true,"dcValue8":";;tessinerstrasse96;rostock;;1XXXX;","dcValue9":";;Tessiner Strasse 96;Rostock;;1XXXX;","dcValue10":";;tessinerstrasse96;rostock;;1XXXX;","dcValue11":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","dcValue12":"XX.XXXXXXX","dcValue13":"XX.XXXXXXXXXXXX","dcValue14":true},{"dcValue1":2,"dcValue2":2,"dcValue3":2,"dcValue4":2,"dcValue5":2,"dcValue6":2,"dcValue7":5,"dcValue8":2,"dcValue9":2,"dcValue10":2,"dcValue11":2,"dcValue12":2,"dcValue13":2,"dcValue14":5}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/DB\/QueryBuilder\/QueryBuilder.php","line":214,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":269,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":110,"function":"scheduleForLookup","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":58,"function":"lookupAddress","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Controller\/ContactsController.php","line":93,"function":"addressToGeo","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":166,"function":"getContacts","class":"OCA\\Maps\\Controller\\ContactsController","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Maps\\Controller\\ContactsController"},"getContacts"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/App.php","line":126,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Maps\\Controller\\ContactsController"},"getContacts"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Routing\/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Maps\\Controller\\ContactsController","getContacts",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"_route":"maps.contacts.getContacts"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"_route":"maps.contacts.getContacts"}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/Route\/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"_route":"maps.contacts.getContacts"}]},{"file":"\/var\/www\/nextcloud\/lib\/base.php","line":975,"function":"match","class":"OC\\Route\\Router","type":"->","args":["\/apps\/maps\/contacts"]},{"file":"\/var\/www\/nextcloud\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Driver\/PDOStatement.php","Line":143,"Previous":{"Exception":"PDOException","Message":"SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'object_uri' at row 1","Code":"22001","Trace":[{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Driver\/PDOStatement.php","line":141,"function":"execute","class":"PDOStatement","type":"->","args":[null]},{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Connection.php","line":1079,"function":"execute","class":"Doctrine\\DBAL\\Driver\\PDOStatement","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/lib\/private\/DB\/Connection.php","line":216,"function":"executeUpdate","class":"Doctrine\\DBAL\\Connection","type":"->","args":["INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(?, ?, ?, ?, ?, ?)",[";;Tessiner Strasse 96;Rostock;;1XXXX;",";;tessinerstrasse96;rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","XX.XXXXXXX","XX.XXXXXXXXXXXX",true],[2,2,2,2,2,5]]},{"file":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Query\/QueryBuilder.php","line":222,"function":"executeUpdate","class":"OC\\DB\\Connection","type":"->","args":["INSERT INTO `oc_maps_address_geo` (`adr`, `adr_norm`, `object_uri`, `lat`, `lng`, `looked_up`) VALUES(:dcValue9, :dcValue10, :dcValue11, :dcValue12, :dcValue13,:dcValue14)",{"dcValue1":"d784cbd5-17a2-4283-96b0-a55ae754694d.vcf","dcValue2":";;th\u00e4lmann-stra\u00dfe;rostock;mecklenburg-vorpommern;1XXXX;deutschland","dcValue3":"59C5806D-156E-4D72-B7D2-48189AB30237.vcf","dcValue4":";;yyy-zzz-stra\u00dfe50;m\u00fcnchen;bayern;8XXXX;deutschland","dcValue5":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","dcValue6":";;tessinerstrasse96;rostock;;1XXXX;","dcValue7":true,"dcValue8":";;tessinerstrasse96;rostock;;1XXXX;","dcValue9":";;Tessiner Strasse 96;Rostock;;1XXXX;","dcValue10":";;tessinerstrasse96;rostock;;1XXXX;","dcValue11":"040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf","dcValue12":"XX.XXXXXXX","dcValue13":"XX.XXXXXXXXXXXX","dcValue14":true},{"dcValue1":2,"dcValue2":2,"dcValue3":2,"dcValue4":2,"dcValue5":2,"dcValue6":2,"dcValue7":5,"dcValue8":2,"dcValue9":2,"dcValue10":2,"dcValue11":2,"dcValue12":2,"dcValue13":2,"dcValue14":5}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/DB\/QueryBuilder\/QueryBuilder.php","line":214,"function":"execute","class":"Doctrine\\DBAL\\Query\\QueryBuilder","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":269,"function":"execute","class":"OC\\DB\\QueryBuilder\\QueryBuilder","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":110,"function":"scheduleForLookup","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Service\/AddressService.php","line":58,"function":"lookupAddress","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/apps\/maps\/lib\/Controller\/ContactsController.php","line":93,"function":"addressToGeo","class":"OCA\\Maps\\Service\\AddressService","type":"->","args":[";;Tessiner Strasse 96;Rostock;;1XXXX;","040000008200E00074C5B7101A82E00800000000CC7902E02AB59E40A6C07553D948A7DC10000000D3E98118F34C6F4C9959C6AC39B14B10.vcf"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":166,"function":"getContacts","class":"OCA\\Maps\\Controller\\ContactsController","type":"->","args":[]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Http\/Dispatcher.php","line":99,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Maps\\Controller\\ContactsController"},"getContacts"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/App.php","line":126,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->","args":[{"__class__":"OCA\\Maps\\Controller\\ContactsController"},"getContacts"]},{"file":"\/var\/www\/nextcloud\/lib\/private\/AppFramework\/Routing\/RouteActionHandler.php","line":47,"function":"main","class":"OC\\AppFramework\\App","type":"::","args":["OCA\\Maps\\Controller\\ContactsController","getContacts",{"__class__":"OC\\AppFramework\\DependencyInjection\\DIContainer"},{"_route":"maps.contacts.getContacts"}]},{"function":"__invoke","class":"OC\\AppFramework\\Routing\\RouteActionHandler","type":"->","args":[{"_route":"maps.contacts.getContacts"}]},{"file":"\/var\/www\/nextcloud\/lib\/private\/Route\/Router.php","line":297,"function":"call_user_func","args":[{"__class__":"OC\\AppFramework\\Routing\\RouteActionHandler"},{"_route":"maps.contacts.getContacts"}]},{"file":"\/var\/www\/nextcloud\/lib\/base.php","line":975,"function":"match","class":"OC\\Route\\Router","type":"->","args":["\/apps\/maps\/contacts"]},{"file":"\/var\/www\/nextcloud\/index.php","line":42,"function":"handleRequest","class":"OC","type":"::","args":[]}],"File":"\/var\/www\/nextcloud\/3rdparty\/doctrine\/dbal\/lib\/Doctrine\/DBAL\/Driver\/PDOStatement.php","Line":141}},"CustomMessage":"--"},"userAgent":"Mozilla\/5.0 (Windows NT 10.0; Win64; x64; rv:68.0) Gecko\/20100101 Firefox\/68.0","version":"16.0.4.1"}
```

Furthermore every access of the maps app shows a message in the top middle:
"Loading of contacts failed"

**Comment by eneiluj** [comment]
@Schmuuu Thanks a lot for this error log pointing to a big bug. A DB field is not big enough for the value we put it in. I'll fix that and it will be included in v0.1.2

**Comment by eneiluj** [comment]
Oh and btw, I don't think german letters can be problematic, they are part of UTF-8.
